src/Cli.Tests/ConfigureOptionsTests.cs

```diff
@@ -109,6 +109,130 @@ public void TestAddDepthLimitForGraphQL()
+        /// in runtime.
+        [DataTestMethod]
+        [DataRow(false, DisplayName = "Update AllowIntrospection to be false for GraphQL.")]
+        [DataRow(true, DisplayName = "Update AllowIntrospection to be true for GraphQL.")]
+        public void TestUpdateAllowIntrospectionForGraphQLSettings(bool updatedValue)
+        {
+            SetupFileSystemWithInitialConfig(INITIAL_CONFIG);
+
+            // Act: Attmepts to update allow-introspection flag
+            ConfigureOptions options = new(
+                runtimeGraphQLAllowIntrospection: updatedValue,
+                config: TEST_RUNTIME_CONFIG_FILE
+            );
+            Assert.IsTrue(TryConfigureSettings(options, _runtimeConfigLoader!, _fileSystem!));
+
+            // Assert: Validate the Allow-Introspection value is updated
+            string updatedConfig = _fileSystem!.File.ReadAllText(TEST_RUNTIME_CONFIG_FILE);
+            Assert.IsTrue(RuntimeConfigLoader.TryParseConfig(updatedConfig, out RuntimeConfig? runtimeConfig));
+            Assert.IsNotNull(runtimeConfig.Runtime?.GraphQL?.AllowIntrospection);
+            Assert.AreEqual(updatedValue, runtimeConfig.Runtime.GraphQL.AllowIntrospection);
+        }
+
+        /// <summary>
+        /// Tests that running "dab configure --runtime.graphql" on a config with variuos values results
+        /// in runtime.
+        [DataTestMethod]
+        [DataRow(false, DisplayName = "Update MultipleMutation.Create.Enabled to be false for GraphQL.")]
+        [DataRow(true, DisplayName = "Update MultipleMutation.Create.Enabled to be true for GraphQL.")]
+        public void TestUpdateMultipleMutationCreateEnabledForGraphQLSettings(bool updatedValue)
+        {
+            SetupFileSystemWithInitialConfig(INITIAL_CONFIG);
+
+            // Act: Attmepts to update multiple-mutations.create.enabled flag
+            ConfigureOptions options = new(
+                runtimeGraphQLMultipleMutationsCreateEnabled: updatedValue,
+                config: TEST_RUNTIME_CONFIG_FILE
+            );
+            Assert.IsTrue(TryConfigureSettings(options, _runtimeConfigLoader!, _fileSystem!));
+
+            // Assert: Validate the Multiple-Mutation.Create.Enabled is updated
+            string updatedConfig = _fileSystem!.File.ReadAllText(TEST_RUNTIME_CONFIG_FILE);
+            Assert.IsTrue(RuntimeConfigLoader.TryParseConfig(updatedConfig, out RuntimeConfig? runtimeConfig));
+            Assert.IsNotNull(runtimeConfig.Runtime?.GraphQL?.MultipleMutationOptions?.MultipleCreateOptions?.Enabled);
+            Assert.AreEqual(updatedValue, runtimeConfig.Runtime.GraphQL.MultipleMutationOptions.MultipleCreateOptions.Enabled);
+        }
+
+        /// <summary>
+        /// Tests that running "dab configure --runtime.graphql" on a config with variuos values results
+        /// in runtime.
+        [TestMethod]
+        public void TestUpdateMultipleForGraphQLSettings()
```

Comment:
What is update multiple? fix the name to reflect your test.

Reply:
Multiple means that multiple parameters of GraphQL are being updated.